src/Sylius/Behat/Page/External/PaypalExpressCheckoutPage.php

```diff
@@ -66,21 +74,42 @@ protected function verifyUrl(array $urlParameters = []): void
         }
     }
 
+    /**
+     * @return TokenInterface
+     *
+     * @throws \RuntimeException
+     */
+    private function findAuthorizeToken()
+    {
+        return $this->findToken('authorize');
+    }
+
     /**
      * @return TokenInterface
      *
      * @throws \RuntimeException
      */
     private function findCaptureToken()
+    {
+        return $this->findToken('capture');
+    }
+
+    /**
+     * @param string $name
+     *
+     * @return TokenInterface
+     *
+     */
+    private function findToken($name)
     {
         $tokens = $this->securityTokenRepository->findAll();
 
         foreach ($tokens as $token) {
-            if (strpos($token->getTargetUrl(), 'capture')) {
+            if (strpos($token->getTargetUrl(), $name)) {
                 return $token;
             }
         }
 
-        throw new \RuntimeException('Cannot find capture token, check if you are after proper checkout steps');
+        throw new \RuntimeException("Cannot find $name token, check if you are after proper checkout steps");
```

Comment:
Could you use `sprintf` here to prevent double quotes and variables in it?
```suggestion
        throw new \RuntimeException(sprintf('Cannot find "%s" token, check if you are after proper checkout steps', $name));
```